app/code/Magento/Inventory/Model/StockItemManager.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model;
+
+use Magento\Framework\Model\AbstractExtensibleModel;
+use Magento\Inventory\Model\ResourceModel\SourceItem as SourceItemResourceModel;
+use Magento\InventoryApi\Api\Data\SourceItemInterface;
+use Magento\InventoryApi\Api\Data\StockItemManagerInterface;
+use Magento\InventoryApi\Api\Data\StockItemManagerExtensionInterface;
+/**
+ * {@inheritdoc}
+ *
+ * @codeCoverageIgnore
+ */
+class StockItemManager extends AbstractExtensibleModel implements StockItemManagerInterface
```

Comment:
why do we have StockItemManager which behaves like DataInterface,

**Manager** means to make some actions over some data.